services/github/github-pipenv.service.js

```diff
@@ -147,7 +149,7 @@ class GithubPipenvLockedDependencyVersion extends ConditionalGithubAuthV3Service
     return {
       label: dependency,
       message: version ? addv(version) : ref,
-      color: 'blue',
+      color: version ? pep440VersionColor(version) : 'blue',
     }
```

Comment:
Could we use `renderVersionBadge` here, either with a conditional or ternary operator? For example:
````
return version ? renderVersionBadge({ version, versionFormatter: pep440VersionColor }) 
               : { label: dependency, message: ref, color: 'blue' }
````

Reply:
We'd have to extend `renderVersionBadge()` to also take a custom label. For this badge the label is the name of the locked dependency. Tbh I'm inclined to leave it